pages/decision-reviews/higher-level-review.md

```diff
@@ -104,14 +139,35 @@ If you have evidence to submit, please select another review option.
 
 <li class="process-step list-four">
 
-### Apply
+### Mail, fax, or drop off the form
 
 Fill out the Decision Review Request: Higher-Level Review (VA Form 20-0996). <br>
 <a href="https://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf">Download VA Form 20-0996 (PDF)</a>
 
+You can also ask for a copy of this form at a regional office, or by calling the VA toll-free hotline, 800-827-1000, Monday through Friday, 8:00 a.m. to 9:00 p.m.
+
 **By mail**
 
-Send the completed form to the benefit office that matches the benefit type you selected on the form.
+Send your completed VA Form 20-0996 to the benefit claims intake center that matches the benefit type you selected on the form.
+
+[Find benefit claims intake center address](#Benefit-claims-intake-center-addresses)
+
+**In person**
+<br>
+Bring your completed form to a regional office.
+<br>
+[Find a regional office near you](/find-locations/)
+<br>
+
+**By fax**
+<br>
+Fax your completed form and any supporting documents to 844-531-7818.
```

Comment:
I don't know what the state of faxing is right now... we were all set to remove it - see https://github.com/department-of-veterans-affairs/vets-website/pull/15203 - but then was told to hold off.